spec/lib/lighthouse/veterans_health/client_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'lighthouse/veterans_health/client'
+
+RSpec.describe Lighthouse::VeteransHealth::Client do
+  # setting the client only once for this test set, as it mimics how it's
+  # used in the Sidekiq worker disability_compensation_fast_track_job.rb
+
+  before(:all) do
+    @client = Lighthouse::VeteransHealth::Client.new(12_345)
+  end
+
+  context 'initialization' do
+    describe 'when the caller passes a valid icn' do
+      it 'initializes the client with the icn set' do
+        expect(@client.instance_variable_get(:@icn)).to eq 12_345
+      end
+    end
+  end
+
+  describe '#get_resource' do
+    let(:jwt) { 'fake_client_assurance_token' }
+    let(:jwt_double) { double('JWT Wrapper', token: jwt) }
+    let(:bearer_token_object) { double('bearer response', body: { 'access_token': 'blah' }) }
+
+    context 'valid requests' do
+      let(:generic_response) { { 'status': 200, 'body': { 'generic': 'response' } } }
```

Comment:
```suggestion
      let(:generic_response) { { status: 200, body: { generic: 'response' } } }
```

Reply:
fixed here: https://github.com/department-of-veterans-affairs/vets-api/pull/8558/commits/49862d3a1e41b62ea9088e10b528e90e8338fc45